js/serial_backend.js

```diff
@@ -181,28 +182,31 @@ function onOpen(openInfo) {
         }, 10000);
 
         FC.resetState();
-
+        MSP.listen(update_packet_error);
+        MSPHelper = new MSPHelper();
```

Comment:
Uh oh, this breaks if you connect, disconnect, and then reconnect.
Reason: The `MSPHelper` prototype gets overwritten by the assignment to `MSPHelper`:

```
extensions::uncaught_exception_handler:8 Error in response to serial.connect: TypeError: MSPHelper is not a constructor
    at onOpen (chrome-extension://bghjhhpmeamhngbeonnholbimdejoghm/js/serial_backend.js:186:21)
    at Object.callback (chrome-extension://bghjhhpmeamhngbeonnholbimdejoghm/js/serial.js:118:31)
```

I'd probably rename the global var into `mspHelper`.

(Minor niggle: For camel case, abbreviations have normally only the first letter capitalised, i.e. `MspHelper`.)